### ISSUE TITLE
Install fails due to missing containeranalysis API

### ISSUE THREAD

**Issue by viglesiasce** [GoogleCloudPlatform/solutions-modern-cicd-anthos#33]
Here is the error: 

```
Step #3 - "create-clusters": Error: Error creating Note: googleapi: Error 403: Container Analysis API has not been used in project 610065905755 before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/containeranalysis.googleapis.com/overview?project=610065905755 then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry.
Step #3 - "create-clusters": 
Step #3 - "create-clusters":   on .terraform/modules/quality-attestor/modules/binary-authorization/main.tf line 50, in resource "google_container_analysis_note" "build-note":
Step #3 - "create-clusters":   50: resource "google_container_analysis_note" "build-note" {
Step #3 - "create-clusters": 
Step #3 - "create-clusters": 
Step #3 - "create-clusters": ```

**Comment by viglesiasce** [comment]
@mike-ensor can you take a look?

**Comment by mike-ensor** [comment]
The timeout to allow the API to be enabled isn't long enough, or more accurately, we need to have a "wait for" command that queries for the API enablement to be propagated.  If re-run, the build will succeed.  I have a 60s wait now and most of the time (4 out of 5 runs) it passed.  I chatted with @morgante yesterday about this.

**Comment by mike-ensor** [comment]
I'll get in a quick-fix to unblock anyone trying to use it, then come back with a more solid fix

**Comment by mike-ensor** [comment]
Just pushed a quick fix, but will create a new issue to track a more comprehensive solution
